### PR TITLE
feat: changes for new common job-manager umbrella and new build and push (MAPCO-4080)

### PR DESCRIPTION
**Pull request by almog8k**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                    |

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -4,7 +4,21 @@ global:
   metrics: {}
   environment: {}
   ingress:
-    domain: 'apps.aroapp.io'  
+    domain: 'apps.aroapp.io'
+  ca:
+    secretName: ''
+    path: '/usr/local/share/ca-certificates'
```
Comment (asafMasa) [inline helm/values.yaml:10]: shouldn't we keep those values empty so they can be overriden from sgared-deployments?

File: helm/values.yaml [hunk 2]
```diff
@@ -28,19 +42,17 @@ authentication:
...
-  db:
-    host: "localhost"
-    name: "raster"
-    port: 5432
-    sslEnabled: false
-    secrets:
-      useExternal: false
-      externalSecretName: ''
+db:
+  host: "localhost"
```
Comment (asafMasa) [inline helm/values.yaml]: shouldn't we keep those values empty so they can be overriden from sgared-deployments?

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: helm/values.yaml (hunk 1 above)
Comment (almog8k) [reply helm/values.yaml:10]: in this case(the ca exist only in global) we don't need to keep the value empty, the umbrella global will override it.

File: helm/values.yaml (hunk 2 above)
Comment (almog8k) [reply helm/values.yaml]: you are right, Done!

File: helm/output [hunk 3]
```diff
@@ -0,0 +1,127 @@
+---
```
Comment (asafMasa) [inline helm/output]: revert

File: output
Comment (asafMasa) [inline output:1]: revert

**Review by asafMasa — CHANGES_REQUESTED** [review]

**Review by asafMasa — APPROVED** [review]

File: .github/workflows/build_and_push.yaml [hunk 4]
```diff
@@ -19,56 +19,10 @@ permissions:
...
-          helm chart save . '${{ secrets.ACR_URL }}/helm/${{ env.CHART }}:${{ env.VER }}'
-        working-directory: ./helm
-      
-      - name: publish chart to acr
-        run: |
-          helm chart push ${{ secrets.ACR_URL }}/helm/${{ env.CHART }}:${{ env.VER }}
-        working-directory: ./helm
+      uses: MapColonies/shared-workflows/.github/workflows/build-and-push-helm.yaml@master
+      secrets: inherit
+
```
Comment (netanelC) [inline .github/workflows/build_and_push.yaml]: Remove these whitespaces

File: .github/workflows/pull_request.yaml [hunk 5]
```diff
@@ -3,81 +3,7 @@ name: pull_request
...
-    steps:
-      - name: Check out Git repository
-        uses: actions/checkout@v2
-
-      - name: build Docker image
-        run: docker build -t test-build:latest .
+  pull_request:
+    uses: MapColonies/shared-workflows/.github/workflows/pull_request.yaml@master
+    secrets: inherit
+
```
Comment (netanelC) [inline .github/workflows/pull_request.yaml]: Remove these whitespaces

File: .github/workflows/release-on-tag-push.yaml [hunk 6]
```diff
@@ -6,33 +6,7 @@ on:
...
-        uses: softprops/action-gh-release@v1
-        with:
-          body_path: CHANGELOG.md
-          files: ${{ env.P_COLLECTION_FILE_NAME }}
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  release_on_tag_push:
+    uses: MapColonies/shared-workflows/.github/workflows/release-on-tag-push.yaml@master
+    secrets: inherit
+
```
Comment (netanelC) [inline .github/workflows/release-on-tag-push.yaml]: Remove these whitespaces

File: helm/templates/_resources.tpl [hunk 7]
```diff
@@ -0,0 +1,34 @@
+{{/*
```
Comment (netanelC) [inline helm/templates/_resources.tpl:1]: Why are you using this file? remove it. It is unnecessary

File: helm/templates/configmap.yaml [hunk 8]
```diff
@@ -1,21 +1,25 @@
...
 {{- $tracingUrl := include "job-manager.tracingUrl" . -}}
 {{- $metricsUrl := include "job-manager.metricsUrl" . -}}
+{{- $db := (include "common.db.merged" .) | fromYaml }}
+
 {{- if .Values.enabled -}}
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Release.Name }}-configmap
+  name: {{ $configmapName }}
```
Comment (netanelC) [inline helm/templates/configmap.yaml:11]: Don't include, just use the chart full name and add "-comfigmap"

File: helm/templates/postgres-secrect.yaml [hunk 9]
```diff
@@ -1,4 +1,5 @@
-{{- if and .Values.local.db.createSecret (not .Values.sharedData.db.secrets.useExternal) -}}
```
Comment (netanelC) [inline helm/templates/postgres-secrect.yaml:1]: Change the name of the file to "secret.yaml"

**Review by netanelC — CHANGES_REQUESTED** [review]
The file resource.tpl is unnecessary.. refactor it

File: helm/templates/configmap.yaml (hunk 8 above)
Comment (almog8k) [reply helm/templates/configmap.yaml:11]: After consulting with @AnguIar and looking threw bitnami implements, we decided to keep this line of method.

File: .github/workflows/build_and_push.yaml (hunk 4 above)
Comment (almog8k) [reply .github/workflows/build_and_push.yaml]: done

File: .github/workflows/pull_request.yaml (hunk 5 above)
Comment (almog8k) [reply .github/workflows/pull_request.yaml]: done

File: .github/workflows/release-on-tag-push.yaml (hunk 6 above)
Comment (almog8k) [reply .github/workflows/release-on-tag-push.yaml]: done

File: helm/templates/_resources.tpl (hunk 7 above)
Comment (almog8k) [reply helm/templates/_resources.tpl:1]: After consulting with @AnguIar and looking threw bitnami implements, we decided to keep this line of method.

File: helm/templates/postgres-secrect.yaml (hunk 9 above)
Comment (almog8k) [reply helm/templates/postgres-secrect.yaml:1]: done

**Review by netanelC — APPROVED** [review]

**Review by ronenkapelian — APPROVED** [review]
good job Boy

**Pull request merged**